scripts/authors.scala

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+exec scala "$0" "$@"
+!#
+
+/*
+ * Shamelessly copied from https://github.com/akka/akka/blob/master/scripts/authors.scala
+ * Usage:
+ *    authors.scala tag1 tag2
+ *
+ * or if on non unixy os:
+ *    scala authors.scala tag1 tag2
+ *
+ * requires scala 2.13.x+ and command line git on path
+ */
+
+import scala.sys.process._
+
+require(args.length == 2, "usage: authors prevTag currTag")
+
+val gitCmd = "git log --no-merges --shortstat -z --minimal -w -C " + args(0) + ".." + args(1)
```

Comment:
Funny thing is, when you call this git command with the `currTag` first and the `prevTag` second - it still works and returns invalid data (commits before the `prevTag`).

Simple `git tag` may be used to validate if the first tag was before the second.

Reply:
`git log` doesn't seem to care so neither will I, it's just a tool for us